src/lua/compat.h

```diff
@@ -14,7 +14,17 @@ extern "C" {
 
 struct lua_State;
 
+/* toggler of mp_char2escape */
+int
+msgpuck_esc_slash_toggle(struct lua_State *L);
```

Comment:
Here I also don't like 'toogle' name. Something like `lbox_cfg_msgpuck` would be better, IMHO. Maybe it also worth to expose this function (as internal) from `msgpack` built-in module.

Reply:
I don't want to introduce dependency on another patch to a [different repository](https://github.com/tarantool/msgpuck/).

Reply:
Also, we don't change msgpuck cfg with the toggler, I am even not sure if there is one. Thus, the naming is weird, `lbox_msgpuck_esc_slash()` would be better, but it doesn't seem too different from the current.